instrumentation/instasarama/consumer_group_handler_test.go

```diff
@@ -24,13 +24,24 @@ func TestConsumerGroupHandler_ConsumeClaim(t *testing.T) {
 			Topic: "topic-1",
 			Headers: []*sarama.RecordHeader{
 				{
+					Key:   []byte("x_instana_t"),
+					Value: []byte("0000000000000000000000000abcde12"),
+				},
+				{
+					Key:   []byte("x_instana_s"),
+					Value: []byte("00000000deadbeef"),
+				},
+				{Key: []byte("x_instana_l_s"), Value: []byte("1")},
+				{
+					// We deliberately send a different trace and span id in the binary header as in the string header to validate
+					// that the string headers get preference when both formats are present in the incoming message.
```

Comment:
Not sure if this is a valid way to test this, please double check.

Reply:
Looks fine for me.